charter.md

```diff
@@ -0,0 +1,55 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established. By default, the group will operate in public. However, specific decision-making processes can be made member-confidential by defining them as such in a process document published with the consensus of the team.
+
+The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus at a minimum once per calendar year. The term limit for the chair will be 1 year and the same individual can be reappointed as chair, if there is consensus. If the team is unable to meet due to absence of a chair, the Core Team may appoint one directly.
+
+Meeting time and agenda should be announced by the chair at least 48 hours in advance, and meeting minutes posted in the repository.
+
+The Interop Team's work is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+All contributions are licensed under the terms in [CONTRIBUTING.md](https://github.com/web-platform-tests/wpt/blob/master/CONTRIBUTING.md).
+
+Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs).
+
+## Scope
+
+The Interop Team's scope includes metrics designed to measure and improve the interoperability of the web platform. These metrics are published once consensus to do so has been reached.
+
+These metrics are expected to be comprised of:
+
+- Groups of automated tests that can be used to automatically compute an interoperability score for specific platform features in implementations.
+- Work to improve the interoperability of the platform that cannot be measured through test pass rates and is scored by consensus decision.
+
+The team is responsible for maintaining [Interop 2021](https://wpt.fyi/interop-2021), [2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and defining future interoperability efforts and metrics.
+
+If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.
```

Comment:
```suggestion
If the team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the team's functions revert to the Core Team and this charter is annulled.
```

We otherwise use "Interop Team" and "team", thus be consistent and use lowercase here.